manager/integration/tests/common.py

```diff
@@ -83,7 +83,7 @@
     "storage-over-provisioning-percentage"
 SETTING_STORAGE_MINIMAL_AVAILABLE_PERCENTAGE = \
     "storage-minimal-available-percentage"
-DEFAULT_DISK_PATH = "/var/lib/rancher/longhorn"
+DEFAULT_DISK_PATH = "/var/lib/rancher/longhorn/"
```

Comment:
Why is the ending slash needed?

Reply:
when we run one single test in `test_node`, we won't call `reset_disks_for_all_nodes()` and the default disk path contains the ending slash, then the check like here will fail: https://github.com/shuo-wu/longhorn-tests/blob/94c3a33b857d710f166a5a501462d06f5efb6323/manager/integration/tests/test_node.py#L313